infrastructure/tfvars.json

```diff
@@ -1,28 +1,29 @@
 {
   "location": "West Europe",
-  "terraform_state_storage_account": "dc00tfsa",
-  "terraform_state_storage_container": "dc00tfsc",
+  "terraform_state_storage_account": "agidtfsaprd",
```

Comment:
these IDs are very cryptic :D better naming?

Reply:
some services (ie. storageaccount) require all lowercase names. can you suggest some naming scheme ? =)

Reply:
there's some suggestions [here](https://docs.microsoft.com/en-us/azure/architecture/best-practices/naming-conventions)